std-bits/base/src/main/java/org/enso/base/statistics/StandardDeviation.java

```diff
@@ -0,0 +1,12 @@
+package org.enso.base.statistics;
+
+public class StandardDeviation extends Variance {
+  public StandardDeviation(boolean population) {
+    super(population);
+  }
+
+  @Override
+  public double evaluate(long n, double[] sums) {
+    return Math.sqrt(super.evaluate(n, sums));
+  }
+}
```

Comment:
I don't think inheritance is the right approach here. It breaks the Liskov Substitution Principle (I'm not saying it always has to hold, but here I think it indeed suggests something is wrong).

I suggest to use composition instead of inheritance here, much like in the other cases.